debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/SnapshotReaderIT.java

```diff
@@ -557,14 +586,10 @@ public void shouldSnapshotTablesInLexicographicalOrder() throws Exception {
         assertArrayEquals(tablesInOrder.toArray(), tablesInOrderExpected.toArray());
     }
 
-    private Function<SourceRecord, String> getTableNameFromSourceRecord = sourceRecord -> ((Struct) sourceRecord.value()).getStruct("source").getString("table");
+    private final Function<SourceRecord, String> getTableNameFromSourceRecord = sourceRecord -> ((Struct) sourceRecord.value()).getStruct("source").getString("table");
 
     private LinkedHashSet<String> getTableNamesInSpecifiedOrder(String... tables) {
-        LinkedHashSet<String> tablesInOrderExpected = new LinkedHashSet<>();
-        for (String table : tables) {
-            tablesInOrderExpected.add(table);
-        }
-        return tablesInOrderExpected;
+        return new LinkedHashSet<>(Arrays.asList(tables));
```

Comment:
Better to leave these kinds of changes for a separate clean-up commit. You can do `git add --patch` in order to isolate them.